tasks/configure_agent.yml

```diff
@@ -5,7 +5,7 @@
     state: "{{ nrinfragent_state }}"
   when:
     - nrinfragent_state != "absent"
-    - nrinfragent_os_name != 'windows'
+    - nrinfragent_os_name|lower != 'windows'
```

Comment:
@JakubKotkowiak the reason I had removed all the `|lower`s both here and below is that I put these in the `var/main.yml`.  Do you think it is still necessary to use the `|lower` in additional places?

Reply:
Ah, I've overlooked it. In that case I will fix it to not use lower

Reply:
OK